channels/cu.m3u

```diff
@@ -1,3 +1,13 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="CubaVision.cu" tvg-name="CubaVision" tvg-country="CU" tvg-language="" tvg-logo="" group-title="",CubaVision
-http://190.122.96.187:8888/http/010
```

Comment:
I'm sorry, but I don't understand why you deleted this link?

Reply:
Not working.

Reply:
Well, it seems to be working now.
![image](https://user-images.githubusercontent.com/7253922/121071857-29ea8c80-c7d9-11eb-88a5-dc06c1bbfb6f.png)

Reply:
Reintegrating it, changing its name since it's the International version.